chat-components/src/main/java/com/pubnub/components/chat/ui/component/input/TypingIndicator.kt

```diff
@@ -0,0 +1,35 @@
+package com.pubnub.components.chat.ui.component.input
+
+import androidx.compose.foundation.layout.Column
+import androidx.compose.foundation.layout.ColumnScope
+import androidx.compose.runtime.Composable
+import androidx.compose.runtime.DisposableEffect
+import androidx.compose.runtime.collectAsState
+import androidx.compose.runtime.getValue
+import com.pubnub.components.chat.provider.LocalMemberFormatter
+import com.pubnub.components.chat.ui.component.provider.LocalChannel
+import com.pubnub.components.chat.ui.mapper.typing.DomainTypingMapper
+import com.pubnub.framework.service.LocalTypingService
+
+@Composable
+fun TypingIndicator(
+    typingIndicatorContent: @Composable ColumnScope.(List<TypingUi>) -> Unit = { typing ->
+        TypingIndicatorContent(typing)
+    }
+) {
+    Column {
+        checkNotNull(LocalChannel.current)
```

Comment:
`channel` can be extracted as a parameter to allow passing channelId to component